arviz/tests/base_tests/test_plots_matplotlib.py

```diff
@@ -107,6 +107,25 @@ def data_list():
     return list(range(11, 31))
 
 
+@pytest.mark.parametrize(
```

Comment:
these kwargs don't affect the behaviour of `filter_vars` so they should be removed to keep tests as short and to the point as possible. See also comment below.